src/Nethermind/Nethermind.Runner/configs/op-mainnet_archive.json

```diff
@@ -10,11 +10,6 @@
   "TxPool": {
     "BlobsSupport": "Disabled"
   },
-  "Sync": {
-    "NetworkingEnabled": false,
-    "AncientBodiesBarrier": 105235063,
```

Comment:
Barrier not needed anymore?

Reply:
No - especially on archive config xD Not sure why it was even there

Reply:
Actually learnt today about Regenesis of op-mainnet.
So reverted barriers